hatice-ecevit-2016400138/assignment1/Anagram.java

```diff
@@ -63,6 +63,8 @@ public static boolean areAnagrams(String s1, String s2, boolean caseSensitive){
     public static boolean areAnagramSentences(String s1,String s2, boolean caseSensitive){
         String[] words1 = s1.split("\\s+");
         String[] words2 = s2.split("\\s+");
+        Boolean[] isMatched1 = new Boolean[words1.length];
+        Boolean[] isMatched2 = new Boolean[words2.length];
         if (words1.length != words2.length) {
             return false;
         }
```

Comment:
Optional: I think it would simplify the code structure to drop the `else` if the first part is a simple return.
Essentially, as a reader, I do not have to keep in memory what is the condition of this else.

What I mean is that:

```java
  if (words1.length != words2.length) {
    return false;
  }

  boolean[] anagram1 = new boolean[words1.length];
  boolean[] anagram2 = new boolean[words2.length];
  ...
```

is easier to read that having the `else` introduce a nested block.